guides-markdown/multisig-cli.md

```diff
@@ -0,0 +1,463 @@
+  "keys": [
+    "tpubDCGKejeBy1oMoZ57NvG2u1X7pffLHkKxew1RAohEzaoPxSSXPdy7st8caWdwymqStLSXxsC4cvKcm9FSLS2PNEcS5rFHo5ccL9xCecqkQth",
+    "tpubDDGK1KoY3YQ1xSwz4AyigwWPickr4mFVtRxu7ZZMQz5BDFGNF6SkgJq2kEfTzuudZVgy84rMVZeMv8GDVod2AGxgZSy3fBpWHpYHTEdUkZc",
+    "tpubDDpd2qFrbXW35YkebSqvihtEsUW9ynmpy61FeVmquPzYzYmFupmabBnHwhJjJighrt2Cmss2Ndg7SJ1iXLuQvZia8KCRWvPTBEacpx6BARf",
+    "tpubDDtiGfAVwUAz2wHGh24TCsk6smz8QbHLqr7swEjS8YiEm6BczQPjqRVBuEqPefUCLur1U2VB4P8TUXQg9vXqzQdUx8bynP574sbzuADmRff"
+  ],
+  "balance": {
+    "tx": 0,
+    "coin": 0,
+    "unconfirmed": 0,
+    "confirmed": 0
+  }
+}
+```
+
+Note that even though the status indicates `"watchOnly": false`, Alice can not spend
+from this wallet by herself. She only has one private key (and four public keys)
+per address.
+
+Bob, Charlie, David, and Erica all go through the same process of importing the other
+keys. In the end, all five participants will have the same wallet structure, and
+**identical** receive addresses. Assuming each person is running their own bcoin
+full node on their own secure machine, they will now have a totally decentralized
+multi-signature wallet! The participants do not need to trust each other to set
+up or use this scheme. Some participants may even be using a hardware wallet like
+a Ledger, connected to their bcoin full node with the
+[bledger](https://github.com/bcoin-org/bledger) package.
+
+## Let's spend some money (as a team!)
+
+So now we have five wallets, each with one unique private key, and four shared
+public keys. All five wallets show the same receive address. Let's fund it!
+The easiest way to do that in regtest mode is by generating blocks:
+
+```
+$ bcoin-cli rpc generatetoaddress 100 bcrt1q59779rsw4267cfts6f0q2nhth49casntju7gnf6n52uj75ttzwys3zph8r
+```
+
+All five wallets will now show the same balance. Here's Bob & Charlie as examples:
+
+```
+$ bwallet-cli --id=Bob balance
+{
+  "account": -1,
+  "tx": 100,
+  "coin": 100,
+  "unconfirmed": 500000000000,
+  "confirmed": 500000000000
+}
+$ bwallet-cli --id=Charlie balance
+{
+  "account": -1,
+  "tx": 100,
+  "coin": 100,
+  "unconfirmed": 500000000000,
+  "confirmed": 500000000000
+}
+```
+
+So, I am the host of this party and I need everyone to chip in for the pizza.
+Here's my personal wallet address: `mpEwzcAKTsuGMM7hV49Mh3Ge7r933S6uYd`, and I
+need 1.23456789 BTC from the group. This is where it gets interesting:
+
+- ANY single participant can create a transaction spending from the multisig address.
+This transaction will be created and signed by that first participant. This is often
+called a "proposal" in multisig wallet schemes.
+
+- This transaction with its single signature is NOT valid, and will be rejected
+from the Bitcoin network.
+
+- Because our scheme is 3-of-5, we only need to get two other participants to add
+their signatures before the transaction is valid.
+
+- Although bcoin does not yet currently comply with the proposed standard
```

Comment:
typo: "does not yet currently comply" is redundant. We should get rid of "yet" or "currently."